### ISSUE TITLE
Something wrong of the training result? 

### ISSUE THREAD

**Issue by cr496352127** [2018110060ding/EmotionCaps#2]
I try to process this demo. However, the is something wrong in the result. 
Mon Nov  1 22:22:37 2021
Namespace(batch_size=100, debug=0, epochs=40, gpus=2, lam_regularize=0.0, lr=1e-05, routings=3, save_dir='/root/result/result_deap/sub_dependent_v2/', testing=False, weights=None)
training examples: 2160
test examples    : 240
__________________________________________________________________________________________________
Layer (type)                    Output Shape         Param #     Connected to                     
==================================================================================================
input_5 (InputLayer)            (None, 128, 32, 1)   0                                            
__________________________________________________________________________________________________
conv1 (Conv2D)                  (None, 120, 24, 256) 20992       input_5[0][0]                    
__________________________________________________________________________________________________
primarycap_conv2d (Conv2D)      (None, 120, 24, 256) 5308672     conv1[0][0]                      
__________________________________________________________________________________________________
concatenate (Concatenate)       (None, 120, 24, 512) 0           conv1[0][0]                      
                                                                 primarycap_conv2d[0][0]          
__________________________________________________________________________________________________
bottleneck_layer (Conv2D)       (None, 120, 24, 256) 131328      concatenate[0][0]                
__________________________________________________________________________________________________
primarycap_reshape (Reshape)    (None, 92160, 8)     0           bottleneck_layer[0][0]           
__________________________________________________________________________________________________
primarycap_squash (Lambda)      (None, 92160, 8)     0           primarycap_reshape[0][0]         
__________________________________________________________________________________________________
digitcaps (CapsuleLayer)        (None, 2, 16)        23592960    primarycap_squash[0][0]          
__________________________________________________________________________________________________
capsnet (Length)                (None, 2)            0           digitcaps[0][0]                  
==================================================================================================
Total params: 29,053,952
Trainable params: 29,053,952
Non-trainable params: 0
__________________________________________________________________________________________________
Epoch 1/40
2160/2160 [==============================] - 61s 28ms/step - loss: nan - acc: 0.3324
Epoch 2/40
2160/2160 [==============================] - 56s 26ms/step - loss: nan - acc: 0.3241
Epoch 3/40
2160/2160 [==============================] - 57s 26ms/step - loss: nan - acc: 0.3241
Epoch 4/40
2160/2160 [==============================] - 56s 26ms/step - loss: nan - acc: 0.3241
Epoch 5/40
2160/2160 [==============================] - 56s 26ms/step - loss: nan - acc: 0.3241
Epoch 6/40
2160/2160 [==============================] - 56s 26ms/step - loss: nan - acc: 0.3241
Epoch 7/40
2160/2160 [==============================] - 56s 26ms/step - loss: nan - acc: 0.3241
Epoch 8/40
2160/2160 [==============================] - 56s 26ms/step - loss: nan - acc: 0.3241
Epoch 9/40
2160/2160 [==============================] - 57s 26ms/step - loss: nan - acc: 0.3241
Epoch 10/40
2160/2160 [==============================] - 57s 26ms/step - loss: nan - acc: 0.3241
Epoch 11/40
2160/2160 [==============================] - 56s 26ms/step - loss: nan - acc: 0.3241
Epoch 12/40
2160/2160 [==============================] - 56s 26ms/step - loss: nan - acc: 0.3241
Epoch 13/40
2160/2160 [==============================] - 56s 26ms/step - loss: nan - acc: 0.3241
Epoch 14/40
2160/2160 [==============================] - 57s 26ms/step - loss: nan - acc: 0.3241
......
Environment: Python 3.6, tensorflow-gpu 1.13.1, keras 2.2.4
subject: DEAP dataset - s11
How this problem occurs and how can I solve it? Thanks!
